README.md

```diff
@@ -117,6 +117,27 @@ typeformEmbed.makePopup(url, options)
   )
   ```
 
+### `onSubmit` event
+
+Callback function `onSubmit` receives `event` object with next properties:
+
+| Parameter       | Type   | Description          |
+| --------------- | ------ | -------------------- |
+| `submission_id` | string | ID of the submission |
```

Comment:
This does not seem to match with the code. Its named `response_id` there.

Reply:
good catch 👍